Patienten- &  Besuchskontext PTDATA-1507

simoneOnFhir opened the pull request:
Kapitel überarbeitet, modul-neutral umformuliert und von Dokumentenaustausch nach Basis umgezogen

simoneOnFhir committed:
Create Patient-Fall-Kontext

simoneOnFhir committed:
Rename Patient-Fall-Kontext to Patient-Fall-Kontext.md

Rename file

simoneOnFhir committed:
Update Patient-Fall-Kontext.md

Kapitel überarbeitet und modul-neutral formuliert

simoneOnFhir committed:
Update toc.yaml

Seite in toc hinzugefügt

simoneOnFhir committed:
Rename Patient-Fall-Kontext.md to Patient-Besuch-Kontext.md

rename page

simoneOnFhir committed:
auto-generated FHIR files by GitHub Actions (CI FSH to FHIR Validation)

f-peverali commented:
Habe das hier auf draft gesetzt, da hier noch ein PR auf diesen branch angesetzt : https://github.com/gematik/spec-ISiK-Basismodul/pull/543

simoneOnFhir committed:
Ptdata 1507 fassadenarchitektur (#543)

* Update Patient-Besuch-Kontext.md

Beschreibung der Fassadenarchitektur hinzugefügt.

* Update Patient-Besuch-Kontext.md

Kapitel strukturiert und Best Practice Empfehlung hinzugefügt

* Update Patient-Besuch-Kontext.md

Fix: Tabellenformatierung

* Update guides/Implementierungsleitfaden-ISiK-Basismodul-401/Einfuehrung/UseCasesAnwendung/Patient-Besuch-Kontext.md

* Update guides/Implementierungsleitfaden-ISiK-Basismodul-401/Einfuehrung/UseCasesAnwendung/Patient-Besuch-Kontext.md

---------

Co-authored-by: f-peverali <112709306+f-peverali@users.noreply.github.com>

f-peverali commented:
todo: releasenote

f-peverali committed:
update pics

f-peverali committed:
add releasenotes

f-peverali committed:
Merge branch 'TC_4.0.2' of https://github.com/gematik/spec-ISiK-Basismodul into patienten-fall-kontext

patrick-werner commented on guides/Implementierungsleitfaden-ISiK-Basismodul-401/Einfuehrung/UseCasesAnwendung/Patient-Besuch-Kontext.md:
```diff
@@ -0,0 +1,42 @@
+Dabei ist es den Herstellern von nicht-patientenführenden Systemen ("Subsystemen") freigestellt, ob sie dauerhaft eine vollständige Kopie der relevanten Patienten- und Besuchsdaten des patientenführenden Systems vorhalten wollen. 
+Dies kann zum Beispiel über folgende Mechanismen erfolgen:
+* durch die Entgegennahme und Verarbeitung von HL7 V2 ADT-Nachrichten aus dem patientenführenden System
+* mittels zyklischer Ausführung der _history-Interaktion (derzeit in ISiK nicht spezifiziert) 
```
```suggestion
* mittels zyklischer Abfragen des Ressourcentyp- oder Server-Level _history-Endpoints unter Nutzung von "_since"-Parametern (oder "_lastUpdated") können sowohl neue als auch geänderte Patientendaten ermittelt und abgeholt werden (derzeit in ISiK nicht spezifiziert) 
```

f-peverali committed:
Update guides/Implementierungsleitfaden-ISiK-Basismodul-401/Einfuehrung/UseCasesAnwendung/Patient-Besuch-Kontext.md

Co-authored-by: Patrick Werner <pa.f.werner@gmail.com>

f-peverali reviewed (commented):
question/ suggestion: ich sehe diesen Abschnitt eher unter - https://simplifier.net/guide/isik-basis-401/Einfuehrung/UebergreifendeFestlegungen?version=current

was denkst du @nikohl-de ?

nikohl-de commented:
> question/ suggestion: ich sehe diesen Abschnitt eher unter - https://simplifier.net/guide/isik-basis-401/Einfuehrung/UebergreifendeFestlegungen?version=current
> 
> was denkst du @nikohl-de ?

@f-peverali: Ich stimme dir zu. Für mich ist das ebenfalls eher eine übergreifende Festlegung, als ein Use-Case

f-peverali committed:
move page

f-peverali committed:
Merge branch 'patienten-fall-kontext' of https://github.com/gematik/spec-ISiK-Basismodul into patienten-fall-kontext
